server/src/db/migrations/20220203144345_createReviews.cjs

```diff
@@ -0,0 +1,30 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable('reviews', (table) => {
+        table.bigIncrements('id')
+        table.bigInteger('pastaId')
+            .notNullable()
+            .index()
+            .unsigned()
+            .references('pastas.id')
+        table.string('title').notNullable()
+        table.integer('rating').notNullable()
+        table.string('review')
```

Comment:
Something to discuss with your group, we often try to avoid column/table name duplication so that we don't have `review.review`